how-to-guides/apache-to-mongodb.md

```diff
@@ -36,9 +36,7 @@ For simplicity, this article will describe how to set up a one-node configuratio
 
 The MongoDB Output plugin is included in the latest version of Fluentd's deb/rpm package. If you want to use RubyGems to install the plugin, please use `gem install fluent-plugin-mongo`.
```

Comment:
fluent-package no longer includes fluent-plugin-mongo.
We need to revise or remove the former sentence.

Reply:
But it's out of scope in this PR.
Let's fix it in another PR.